external/builder/fixtures_esprima/deadcode-expected.js

```diff
@@ -0,0 +1,13 @@
+function f1() {
+}
+function f2() {
+ return 1;
+}
+function f3() {
+ var i = 0;
+ throw "test";
+}
+function f4() {
+ var i = 0;
+}
+
```

Comment:
Nit: can we remove the newline at the end since other files also do not have it?